src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -24,7 +24,7 @@ REM Config Certs
 REM we need the direcotry to save the test.crl file. We are investigate a way to get rid of it
 REM
 md c:\wcftest
-pushd ..\..\..\..\bin\Wcf\tools\CertificateGenerator
+pushd %~dp0..\..\..\..\bin\Wcf\tools\CertificateGenerator
 call CertificateGenerator.exe
```

Comment:
I thought it's the best practice to go to that directory before calling a script/tool. If it's not needed, we can remove the push/pop and thus reduce the lines:-)